src/test/kotlin/com/terraformation/backend/db/DatabaseBackedTest.kt

```diff
@@ -1120,6 +1122,25 @@ abstract class DatabaseBackedTest {
     return row.id!!.also { inserted.submissionDocumentIds.add(it) }
   }
 
+  fun insertSubmissionSnapshot(
+      id: Any? = null,
+      fileId: Any? = inserted.fileId,
+      submissionId: Any? = inserted.submissionId,
+  ): SubmissionSnapshotId {
+    nextSubmissionNumber++
```

Comment:
Why increase the submission number here?

Reply:
Thanks for the catch, this was a copy paste failure.